features/src/test/scala/com/salesforce/op/features/types/FeatureTypeSparkConverterTest.scala

```diff
@@ -52,6 +52,12 @@ class FeatureTypeSparkConverterTest
   )
   val bogusNames = Gen.alphaNumStr
 
+  val naturalNumbers = Table("NaturalNumbers", Byte.MaxValue, Short.MaxValue, Int.MaxValue, Long.MaxValue)
```

Comment:
Are you sure that this generates numbers in these ranges correctly? what about negatives and special values?